app/views/external_links/_hathi_links.html.erb

```diff
@@ -1,5 +1,4 @@
 <div class="bs-callout bs-callout-warning">
-  <h5 class="record-view-only" id="conveying-meaning-to-assistive-technologies">Full Text available online</h5>
```

Comment:
@nicktiberi can we use this check https://github.com/psu-libraries/psulib_blacklight/blob/main/app/views/catalog/_show_availability.html.erb#L8 instead of deleting?